ciongke/cmd/hook/githubagent.go

```diff
@@ -37,6 +38,10 @@ type GitHubAgent struct {
 
 	BuildRequests  chan<- KubeRequest
 	DeleteRequests chan<- KubeRequest
+
+	// Cache of org members, protected by the lock.
+	orgMembers map[string]bool
+	mut        sync.Mutex
```

Comment:
Yeah, I'm not a huge fan, but it's really really the least important comment/opinion (because it's just an opinion). Go for it